src/app/shared/classes/certificate.ts

```diff
@@ -0,0 +1,10 @@
+export class Certificate {
```

Comment:
@Yuriy-27  you can improve this

if you add access modifier `private`, `public` or `protected` before params in constructor you will get these params automatically inited as class properties.

just try

```
export class Certificate {
    constructor(
    public id: number;
    public image: string;
    public title: string;
) {}
}
```